app/js/controllers/relatedCtrl.js

```diff
@@ -20,25 +21,29 @@ KMCMenu.controller('relatedCtrl', ['$scope',
 		// update entries list
 		$scope.entryListChange = function(){
 			// unselect playlist ID
-			$scope.playerData.config.player.plugins["related"].playlistId = null;
+			$scope.playerData.config.plugins["related"].playlistId = null;
+			$scope.playerData.config.plugins["related"].useContext = false;
 			$scope.playlistId = {id:'',text:''};
 			// set entries list
-			$scope.playerData.config.player.plugins["related"].entryList = $scope.entryList;
+			$scope.playerData.config.plugins["related"].entryList = $scope.entryList;
 		};
 
 		// update playlist ID
 		$scope.playlistIdChange = function(){
 			// unselect entries list
-			$scope.playerData.config.player.plugins["related"].entryList = null;
+			$scope.playerData.config.plugins["related"].entryList = [];
+			$scope.playerData.config.plugins["related"].useContext = false;
 			$scope.entryList = "";
 			// set playlist ID
-			$scope.playerData.config.player.plugins["related"].playlistId = $scope.playlistId.id;
+			$scope.playerData.config.plugins["related"].playlistId = $scope.playlistId.id;
```

Comment:
does this get called when changing the id to empty ? can you save a config with an empty playlist id ?

Reply:
Can’t be empty, the User has to select from given list